spec/rspec/matchers/built_in/include_spec.rb

```diff
@@ -8,14 +43,23 @@ def build_target(hsh)
       hsh
     end
 
+    def use_string_keys?
```

Comment:
IMO this method name is confusing: from the name it's not clear in which context this controls string keys being used.  There are still lots of uses of symbol keys even when this is set to `true`.

How about `use_string_keys_in_failure_message?`

Reply:
I totally agree with you, so I changed to `use_string_keys_in_failure_message?`.